fem/estimators.hpp

```diff
@@ -114,6 +115,7 @@ class ZienkiewiczZhuEstimator : public AnisotropicErrorEstimator
         integ(&integ),
         solution(&sol),
         flux_space(flux_fes),
+        _with_coeff(false),
```

Comment:
It is best to list the initialization of data members in the order they are declared in the class -- otherwise compilers produce warnings about this, e.g. see any of the Linux or Mac builds in Travis CI.

The simple solution is to move the line
```c++
   _with_coeff(false),
```
right after the line
```c++
   flux_averaging(0),
```

Reply:
Sorry to keep bugging you -- I just noticed these warnings in travis while trying to figure out why one of the tests failed. I'll restart the failed test -- the Mac builds fail randomly sometimes.

Reply:
It is my fault. My apologies. I didn't pay close attention to the warnings. It should be fixed now.